apps/frontend/static_src/scss/components/language-selector.scss

```diff
@@ -49,5 +49,11 @@
             background-color: $color--black;
             color: $color--light-grey;
         }
```

Comment:
Small nitpick: suprised the linter didn't suggest having a space between these two?